samples/graph-meeting-notification/nodejs/README.md

```diff
@@ -0,0 +1,127 @@
+
+### Create Azure bot resource
+
+In Azure portal, create a [Azure Bot resource](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication?view=azure-bot-service-4.0&tabs=csharp%2Caadv2).
+
+- Ensure that you've [enabled the Teams Channel](https://docs.microsoft.com/en-us/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)
+
+### Create and install Self-Signed certificate
+
+To include resource data of graph notifications, this Graph API require self-signed certificate. Follow the below steps to create and manage certificate.
+
+1. You can self-sign the certificate, since Microsoft Graph does not verify the certificate issuer, and uses the public key for only encryption.
+
+2. Use [Azure Key Vault](https://docs.microsoft.com/en-us/azure/key-vault/key-vault-whatis) as the solution to create, rotate, and securely manage certificates. Make sure the keys satisfy the following criteria:
+
+    - The key must be of type `RSA`
+    - The key size must be between 2048 and 4096 bits
+
+3. Follow this documentation for the steps - [**Create and install Self-Signed certificate**](CertificateDocumentation/README.md)
+
+### Setup code.
+1) Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+  - Launch Visual Studio code
+  - File -> Open Folder
+  - Navigate to `samples/graph-change-notification-team-channel/nodejs` folder.
+
+  - Update the `.env` file
+
+   Update configuration with the ```MicrosoftAppId```, ```MicrosoftAppPassword```, ```MicrosoftAppTenantId```,
+   ```Base64EncodedCertificate```, ```EncryptionCertificateId```, ```PRIVATE_KEY_PATH```
+
+### Instruction for .env
+1. Provide `MicrosoftAppId`, `MicrosoftAppPassword` and `MicrosoftAppTenantId` in the .env that is created in Azure.
+2. Provide the ngrok url as  `BaseUrl` in appsetting on which application is running on.
+3. You should be having `Base64EncodedCertificate` from *Create and install Self-Signed certificate* step.
+4. Use Certificate "PEM" format and add the certificate name for `PRIVATE_KEY_PATH` For eg  `PRIVATE_KEY_PATH`=PrivateKeyFileName.pem" in .env file. Also make sure the private key file is stored inside helper folder of this project.
+
+ - Install node modules
+
+    Inside node js folder, open your local terminal and run the below command to install node modules. You can do the same for client folder by opening the project in Visual Studio code.
+
+    ```bash
+    npm install
+    ```
+
+- Run your app
+
+   ```bash
+     npm start
+   ``` 
+
+ - Run ngrok - point to port 3978
+
+   ```bash
+     ngrok http -host-header=rewrite 3978
+
+## Instruction for manifest
+1. Fill any GUID for <APP-ID>. You can also put your MicrosoftAppId here.
```

Comment:
Fill any GUID for <GUID> in your manifest. As an alternative, you can also put your Microsoft App Id.